buffer/src/main/java/io/netty/buffer/ByteBufProcessor.java

```diff
@@ -21,11 +21,13 @@
 /**
  * @deprecated Use {@link ByteProcessor}.
  */
+@Deprecated
```

Comment:
@normanmaurer - I don't see why this would be a false positive. `ByteBufProcessor` should be deprecated in favor of `ByteProcessor` and the comments indicate that it is deprecated. Can you elaborate?